### ISSUE TITLE
Cannot find Heartbeat in search results

### ISSUE THREAD

**Issue by DICKR** [GetiPlayerAutomator/get-iplayer-automator#577]
Unable to download anything. The programme opens OK, but on entering a request all I get repeatedly is
NO SHOWS FOUND even after multiple Re boots and hundreds of re instal attempts  Grateful for any help. Can record my beloved HEARTBEATS or anything.

**Comment by dinkypumpkin** [comment]
Don't use your name as the issue title. Use a title that describes the problem. Unless you provide any other examples, this is assumed to be duplicate of #547

**Comment by DICKR** [comment]
Dear Dinkypumpkin,
You asked me to reply to this e mail.  At 88 years old I am frustrated and
I have outlined the problem several times. In layman's language please tell
me what i must do to be able to use I PLAYER AUTOMOTOR without complicated
instructions.
After a long time without this system, I have given up any hope of getting
it back.
Thanks for your interest and advice.
Regards  Harold Reast  B E M

**Comment by larryy** [comment]
Harold (DICKR) - I don't work on Get iPlayer Automator (GiA), but I can tell you that ITV programs, including (my also beloved) Heartbeat, are no longer included in the search results because of a change made by ITV.  Unfortunately, this cannot be fixed from GiA's side.  So nothing will ever be found if you are searching for an ITV program.  You can, however, use GiA's "Use Current Webpage" feature for ITV programs, including Heartbeat.  To do this...

* Find the show you want to record on itv.com in a browser.  (For example, all the current episodes of Heartbeat can be found here: http://www.itv.com/hub/heartbeat/Ya0757  You would click on one of those episodes so the browser was showing you just the episode you want to download.)
* Click "Use Current Webpage" in GiA.  This will add the show to the download queue.
* Start the downloads.

That's it.  Although there are two assumptions here:

1) First, you must have the most recent version of GiA.  That is 1.8.6 at current time.
2) Second, you must make sure GiA knows which browser to look at.  In GiA's preferences you set "Default Browser (for Retrieving PIDs):" to the browser you are using.  I believe it must be either Safari or Chrome.

I know this works because I have been completing my collection of the higher resolution Heartbeat episodes recently.  Unfortunately, they only ever show series 11-18.  I wish they'd go back and show series 1-10.  I've emailed them asking them to do this.  Maybe you could as well, and we'd eventually get to see better quality versions of those early episodes.

**Comment by DICKR** [comment]
Dear ?
Many thanks indeed for the helpful e mail. I already have HEARTBEAT Seasons 1 to 8 or even 9 and 10. I really want any that is still available. Thanks also informing of  ITV's restrictions. Downloaded the app again yesterday,and already tried Use Current Webpage without any results -Will try again now.  Appreciate the patience and help you have given me (I don't know where else I can get help (maybe  U TUBE ?)
Many thanks
Harold Reast (CYPRUS)
